scripts/parse-blacklisted-modules.js

```diff
@@ -5,7 +5,9 @@ const network = process.argv[2]
 const blacklistedModules = require('../blacklisted-modules')[network]
 
 console.log(`Blacklisting modules: ${blacklistedModules}`)
+const outputPath = `${process.cwd()}/helpers/blacklisted-modules.ts`
```

Comment:
```suggestion
const outputPath = path.join(process.cwd(), 'helpers/blacklisted-modules.ts')
```

Nitpick: generally might as well use the built-in path library :).